extensions/wikia/AdEngine/js/lookup/prebid/adapters/aol.js

```diff
@@ -59,6 +59,14 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.aol', [
 					alias: '4431475',
 					sizeId: '529'
 				}
+				// TODO: fill with proper numbers https://wikia-inc.atlassian.net/browse/ADEN-5517
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)